app/controllers/users/friendships_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Users::FriendshipsController < ApplicationController
+  def create
+    invitee = User.find(params[:user])
+    if params[:status] == 'accepted'
+      Friendship.create(user: current_user, friend: invitee, status: :accepted)
+      friendship = Friendship.find_by(user: invitee, friend: current_user)
+      friendship.update(status: :accepted)
```

Comment:
I feel like there could be a way that we don't need two friendships for one relationship

Reply:
I agree!  Probably best to set a method on User model that says friends = friendships.where(user: self || friend: self)